Author: fwang
Subject: Build fix: Fix glib header inclusion
Applied-Upstream: commit:30602
Forwarded: yes
Bug: http://bugzilla.abisource.com/show_bug.cgi?id=13247

--- ./goffice-bits/goffice/app/goffice-app.h.glib	2011-11-25 16:03:34.174789292 +0800
+++ ./goffice-bits/goffice/app/goffice-app.h	2011-11-25 16:03:41.994773077 +0800
@@ -22,7 +22,7 @@
 #ifndef GOFFICE_APP_H
 #define GOFFICE_APP_H
 
-#include <glib/gmacros.h>
+#include <glib.h>
 
 G_BEGIN_DECLS
 
--- ./goffice-bits2/goffice/app/goffice-app.h.glib	2011-11-25 16:03:46.674788343 +0800
+++ ./goffice-bits2/goffice/app/goffice-app.h	2011-11-25 16:03:54.044789574 +0800
@@ -22,7 +22,7 @@
 #ifndef GOFFICE_APP_H
 #define GOFFICE_APP_H
 
-#include <glib/gmacros.h>
+#include <glib.h>
 
 G_BEGIN_DECLS
 
--- ./src/af/util/xp/ut_go_file.h.glib	2011-11-25 16:15:20.746058195 +0800
+++ ./src/af/util/xp/ut_go_file.h	2011-11-25 16:15:27.184659115 +0800
@@ -31,7 +31,6 @@
 
 #include <glib.h>
 #include <gsf/gsf.h>
-#include <glib/gerror.h>
 #include <time.h>
 
 G_BEGIN_DECLS
