--- abiword-3.0.2/plugin-configure.m4.orig	2018-03-19 13:26:04.728789509 +0000
+++ abiword-3.0.2/plugin-configure.m4	2018-03-19 13:26:35.301155743 +0000
@@ -72,7 +72,7 @@
 AC_SUBST([GRAMMAR_LIBS])
 
 
-wpg_pkgs="$gsf_req libwpg-0.2 >= 0.2.0 libwpd-0.9 >= 0.9.0 libwpd-stream-0.9 >= 0.9.0"
+wpg_pkgs="$gsf_req libwpg-0.3 >= 0.2.0 libwpd-0.10 >= 0.9.0 "
 wpg_deps="no"
 
 if test "$enable_wpg" != ""; then
@@ -962,8 +962,8 @@
 AC_SUBST([XSLFO_LIBS])
 
 
-wordperfect_pkgs="libwpg-0.2 >= 0.2.0 libwpd-0.9 >= 0.9.0 libwpd-stream-0.9 >= 0.9.0 $gsf_req"
-wordperfect_wps_pkgs='libwps-0.2 >= 0.1.0'
+wordperfect_pkgs="libwpg-0.3 >= 0.2.0 libwpd-0.10 >= 0.9.0 $gsf_req"
+wordperfect_wps_pkgs='libwps-0.4 >= 0.1.0'
 wordperfect_deps="no"
 
 WORDPERFECT_CFLAGS=
--- a/plugins/wordperfect/xp/ie_imp_WordPerfect.cpp
+++ b/plugins/wordperfect/xp/ie_imp_WordPerfect.cpp
@@ -1389,7 +1389,9 @@ UT_Confidence_t IE_Imp_MSWorks_Sniffer::recognizeContents (GsfInput * input)
 	AbiWordperfectInputStream gsfInput(input);

 	libwps::WPSKind kind;
-	libwps::WPSConfidence confidence = libwps::WPSDocument::isFileFormatSupported(&gsfInput, kind);
+        libwps::WPSCreator creator;
+        bool needsEncoding = false;
+      libwps::WPSConfidence confidence = libwps::WPSDocument::isFileFormatSupported(&gsfInput, kind, creator, needsEncoding);
 	
 	if (kind != libwps::WPS_TEXT)
 		confidence = libwps::WPS_CONFIDENCE_NONE;

